manifests-maistra/1.1.0/servicemeshcontrolplanes.crd.yaml

```diff
@@ -16,25 +16,30 @@ spec:
     status: {}
   version: v1
   additionalPrinterColumns:
-  - JSONPath: .status.conditions[?(@.type=="Ready")].status
-    name: Ready
+  - name: Ready
+    description: How many of the total number of member namespaces are
```

Comment:
Looks like this might be mislabeled.  Also, it seems like it might be better to list the components that are not ready vs a count.  That would allow users to easily see what's not up and running.

Reply:
This follows the pattern used elsewhere - in deployments, where the READY column shows the number of pods, and in pods, where it shows the number of containers that are ready vs. the total number.

We should follow the same pattern in our own resources.

Showing just the number of "unready" components is kinda weird. And it doesn't show how many components there are in total. The ready/total approach gives you more information and doesn't use up any more width, since the column header is wider than the actual value. Also, naming the column "UNREADY" would just make the column wider.

Reply:
Remember, `oc get` should show just the summary info that allows users to quickly see whether everything is OK or not. When something is wrong, users typically use `oc describe` to get further info.